azure/durable_functions/tasks/continue_as_new.py

```diff
@@ -21,4 +21,4 @@ def continue_as_new(
     """
     new_action = ContinueAsNewAction(input_)
 
-    return Task(is_completed=False, is_faulted=False, action=new_action)
+    return Task(is_completed=True, is_faulted=False, action=new_action)
```

Comment:
I also realize that I never addressed https://github.com/Azure/azure-functions-durable-js/issues/126 in Durable JS, so we should do that correct this time.

In fact, since we have a clean state here, my recommendation would be to make this a void method as opposed to returning a Task.

Reply:
Due to how the out-of-proc model works, there's no clean way of allowing the method to work without `yield` while also "forcing it to fail / error out" if yielded. I can go that extra mile though, if necessary.

My latest updates enables continue_as_new to work with and without `yield` :)